ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -578,7 +606,8 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "caseworker-civil-admin"
+          "caseworker-civil-admin",
+          "caseworker-civil-judge"
```

Comment:
This makes sense, but from memory this Case Event was specifically requested for Admins, Judges might not need this - one for the business to confirm :)